predicators/spot_utils/spot_utils.py

```diff
@@ -1257,15 +1288,54 @@ def navigate_to_position(self, params: Array) -> None:
         try:
             # (1) Initialize location
             self.graph_nav_command_line.set_initial_localization_fiducial()
-            self.graph_nav_command_line.graph_nav_client.get_localization_state(
-            )
+            self.graph_nav_command_line.graph_nav_client.\
+                get_localization_state()
 
             # (2) Just move
             self.relative_move(params[0], params[1], params[2])
 
         except Exception as e:
             logging.info(e)
 
+    def drag_arm_control(self, params: Array) -> None:
+        """Simple drag impedence controller."""
```

Comment:
this isn't an impedance controller anymore right? let's update this comment accordingly